backend/inspirehep/mailing/api/seminars.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+import structlog
+from flask import current_app, render_template
+
+from inspirehep.mailing.providers.flask_mail import send_email
+
+LOGGER = structlog.getLogger()
+
+
+def get_submissions_confirmation_email_html_content(seminar):
+    host = current_app.config["SERVER_NAME"]
```

Comment:
```suggestion
    host = get_inspirehep_url()
```

Reply:
full url looks ugly in the email, isn't `https` by default anyhow?

Reply:
it's using https https://github.com/inspirehep/inspirehep/blob/b6aa07abab5504eb5744ef74d161a1f5c1e2f208/backend/inspirehep/utils.py#L18

Reply:
No no I mean, the load balancer.. it's routing to `https` by default and `protocol` seems ugly in the mail content.